tests/functional/backend/corpora/test_revisions.py

```diff
@@ -166,10 +166,11 @@ def test_revision_flow(self):
             res = requests.get(f"{self.api}/dp/v1/datasets/meta?url={self.create_explorer_url(deleted_dataset_id)}")
```

Comment:
I dont think this test should work/if it is working we may an issue. This endpoint (`/datasets/meta`) shouldnt be 404ing for a (previously published) deleted dtaset. It should just have the tombstone status set to true and no s3_uri. To check that the dataset was removed from the data portal we probably want to check the collection details endpoint and make sure the dataset is no longer

Reply:
You're right, but I think this test passes nevertheless. Let me verify that and I will add a comment.